stock_outgoing_shipment_report/models/delivery_carrier.py

```diff
@@ -12,3 +12,8 @@ class DeliveryCarrier(models.Model):
         inverse_name="carrier_id",
         string="Available Service(s)",
     )
+
+    ship_billing = fields.Selection(
+        [("third_party", "Third Party"), ("bill_third_party", "Bill Third Party")],
+        string="Ship Billing",
```

Comment:
Add help: `help="The value set here will show as ShipBilling in the outgoing shipment report."`

Reply:
Added it. Thank you.